.buildkite/ftr_configs.yml

```diff
@@ -0,0 +1,600 @@
+disabled:
```

Comment:
This file doesn't exist anymore as of #187440. I suspect this PRs' code has been branched off of main before that change. However, this file should not be brought back.

Reply:
Thanks @pheyos for your feedback. I have made the required changes and re-requested your review.